### DISCUSSION TITLE
Lab_Composition.confidentiality - будем ли писать рекомендации по заполнению?

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#257] [Лабораторная РГ]
Composition.confidentiality (R4) имеет тип code, со следующим [списком значений](https://terminology.hl7.org/3.1.0/ValueSet-v3-Confidentiality.html):
- low
- moderate
- normal
- restricted
- unrestricted
- very restricted

Есть ли российский аналог такого списка? Есть ли необходимость давать рекомендации разработчику по использованию кодов в этом списке?

**Comment by Vrodvi** [comment]
В СЭМДах используется вот это:
https://nsi.rosminzdrav.ru/dictionaries/1.2.643.5.1.13.13.99.2.285/passport/1.2 "Уровень конфиденциальности медицинского документа"

**Reply by eugenekogan-spb** [reply]
Ситуация выглядит аналогичной справочнику полов.

Если действовать аналогично, то мы:
- не профилируем этот атрибут
- рекомендуем разработчикам при использовании набора значений FHIR использовать только некоторые позиции, например unrestricted, restricted, very restricted
- делаем набор значений в соответствии с НСИ МЗ, и concept map, связывающий набор значений FHIR и набор значений НСИ МЗ

Верно?
